### DISCUSSION TITLE
Meeting Data / Image not displaying

### DISCUSSION THREAD

**Discussion by aamonterey** [code4recovery/12-step-meeting-list#1189] [General]
Hello - Can you tell me if the Meeting Data - Image link in the backend is for displaying an image on the front end?

Assuming so, we want to provide an image to help newcomers/visitors find a particular meeting spot. We've uploaded the image to the following meeting, but it's not displaying on the front end: https://www.aamonterey.org/meetings/big-sur-sunday/

Very much appreciate some help with sorting this out.

Thanks - WebAdmin AAMonterey.org

**Comment by brianw-area24** [comment]
Can you post an screen capture/image of what you're looking at on the edit screen, I want to make sure we're looking at the right thing in order to try to help you out.
Thanks,
Brian

**Reply by aamonterey** [reply]
Hi Brian - Please see the attached screenshot. Thanks - Mike

![aamonterey image issue](https://github.com/code4recovery/12-step-meeting-list/assets/87451879/5e031879-2280-4c85-9dd4-fdd0a040a3c7)

**Reply by brianw-area24** [reply]
Thanks for the response. This is a customization that has been made on your web site, and is not part of they TSML plugin. My guess is the legacy meeting details page has also been customized on your site. However, the newer TSML_UI front-end would not have anyway to know about these customizations, therefore the image won't show up when using TSML_UI.

As I'm typing this message, I've been having a side conversation with Josh, and we will bring this at a future meeting of the C4R to see if this is something we think we should include in a future release

**Reply by aamonterey** [reply]
I recall having our developer remove the customization 2 or 3 years ago because it wasn't compatible with some of the TSML updates. Apparently all the custom code was not removed, but TSML has been stable for us every since. 

Appreciate you helping us with this and apologies for taking your time with something that wasn't an issue. All the best!
